format/spec.md

```diff
@@ -631,6 +632,29 @@ When expiring snapshots, retention policies in table and snapshot references are
     2. The snapshot is not one of the first `min-snapshots-to-keep` in the branch (including the branch's referenced snapshot)
 5. Expire any snapshot not in the set of snapshots to retain.
 
+#### Statistics file
+
+Statistics files are valid [Puffin files](../puffin-spec). Statistics are informational. A reader can choose to
+ignore statistics information. Statistics support is not required to read the table correctly.
+
+Statistics files' metadata within `statistics` table snapshot field is a struct with the following fields:
+
+| v2         | Field name                      | Type                                   | Description                                                                                            |
+|------------|---------------------------------|----------------------------------------|--------------------------------------------------------------------------------------------------------|
+| _required_ | **`statistics-path`**           | `string`                               | Path of the statistics file. See [Puffin file format](../puffin-spec).                                 |
+| _required_ | **`file-size-in-bytes`**        | `long`                                 | Size of the statistics file.                                                                           |
+| _required_ | **`file-footer-size-in-bytes`** | `long`                                 | Size of the statistics file's footer. See [Puffin file format](../puffin-spec) for footer definition.  |
+| _required_ | **`source-sequence-number`**    | `long`                                 | Table sequence number at which the stats were calculated                                               |
+| _required_ | **`statistics-metadata`**       | `list<statistic metadata>` (see below) | A list of the statistics metadata for statistics contained in the file with structure described below. |
+
+Statistic metadata is a struct with the following fields:
+
+| v2         | Field name           | Type                  | Description                                                                                      |
+|------------|----------------------|-----------------------|--------------------------------------------------------------------------------------------------|
+| _required_ | **`statistic-type`** | `string`              | Type of the statistic. Matches Blob type in the Puffin file.                                     |
```

Comment:
Why not just `type` or `blob-type`? I think this structure is probably going to be reused for indexes, so making it generic seems like a good idea to me. This is basically `type`, `fields`, and `properties` from the Puffin `BlobMetadata`.

Reply:
cool, will go with `type`